torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -7,7 +7,18 @@
 from torchvision.transforms import functional_pil as _FP, functional_tensor as _FT
 
 
-get_dimensions_image_tensor = _FT.get_dimensions
+def get_dimensions_image_tensor(image: torch.Tensor) -> List[int]:
+    chw = list(image.shape[-3:])
+    ndims = len(chw)
+    if ndims == 3:
+        return chw
+    elif ndims == 2:
+        chw.insert(0, 1)
+        return chw
+    else:
+        raise TypeError("Invalid input tensor dimensions.")
```

Comment:
Can we make this a little more precise? Invalid has a pretty broad meaning and we want to only state that the input has to have at least two dimensions.

Reply:
The previous message on V1 was `Tensor is not a torch image.`. I wanted to avoid it because the kernels are reused for images/videos/masks etc. If you have a better proposal, I'm happy to adopt it.

Reply:
```suggestion
        raise TypeError(f"Input tensor should have at least two dimensions, but got {ndims}")
```